### PR TITLE
Dev.opera blogpost for Opera 51

### PR DESCRIPTION
**Pull request by fsoder**

### CODE REVIEW

File: src/blog/_posts/2018-01-31-opera-51.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+---
+title: What’s new in Chromium 64 and Opera 51
+authors:
+- fredrik-soderqvist
+intro: 'Opera 51 (based on Chromium 64) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To find out what’s new for users, see our [Desktop blog post](http://blogs.opera.com/desktop/2018/01/update-me-opera-51/). Here’s what it means for web developers.'
```
Comment (bratell-at-opera) [inline src/blog/_posts/2018-01-31-opera-51.md]: TODO: Update the link

File: src/blog/_posts/2018-01-31-opera-51.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+- html
+- css
+- dom
+- javascript
+- opera
+license: cc-by-3.0
+---
+
+Opera 51 (based on Chromium 64) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To
+find out what’s new for users, see our [Desktop blog post](http://blogs.opera.com/desktop/2018/01/update-me-too-opera-51/).
```
Comment (bratell-at-opera) [inline src/blog/_posts/2018-01-31-opera-51.md]: TODO: Update the link

File: src/blog/_posts/2018-01-31-opera-51.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+
+## Mitigations for the *"Spectre"* vulnerability
+
+In order to [mitigate](http://blogs.opera.com/security/2018/01/opera-mitigates-critical-cpu-vulnerabilities/) the
+recently disclosed security vulnerability called *"Spectre"*, the following restrictions that affect the functionality
+of the web platform have been applied:
+
+* The `SharedArrayBuffer` functionality has been removed.
+* The resolution of `performance.now()` has been clamped to 100us.
```
Comment (bratell-at-opera) [inline src/blog/_posts/2018-01-31-opera-51.md]: μs, not us.

File: src/blog/_posts/2018-01-31-opera-51.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+Both of the above serve to reduce the possibility of performing a side-channel attack to read sensitive user data, by
+removing potential ways to acquire a high-precision timing source.
+
+## Resize Observer
+
+Responsive web applications are often written using CSS media queries or `window.onresize` to
+build responsive components that adapt content to different viewport sizes. Both of these
```
Comment (bratell-at-opera) [inline src/blog/_posts/2018-01-31-opera-51.md]: Too many "responsive" in one sentence?

File: src/blog/_posts/2018-01-31-opera-51.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+  [exposes](https://googlechromelabs.github.io/web-audio-samples/audio-worklet/) low-level audio processing capability
+  to support custom AudioNodes, is now available behind the experimental flag.
+
+### Blink > WebRTC
+
+* The [`RTCPeerConnection` interface](https://w3c.github.io/webrtc-pc/#interface-definition) now supports `addTrack()`
+  for single stream use cases, as well as `removeTrack()`, `getSenders()` and `ontrack`. A minimal version of the
+  `RTCRtpSender` interface is now also supported.
+
+### Blink > WindowDialog
```
Comment (bratell-at-opera) [inline src/blog/_posts/2018-01-31-opera-51.md]: I guess these are crbug components but they work as titles as well, except maybe this one.

Comment (fsoder) [reply src/blog/_posts/2018-01-31-opera-51.md]: Renaming to "Window APIs"

File: src/blog/_posts/2018-01-31-opera-51.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+  by Opera, like a negative value.
+* [`getUserMedia()`](https://w3c.github.io/mediacapture-main/#dom-mediadevices-getusermedia()) now returns a rejected
+  Promise with a `DOMException` or `OverconstrainedError` when there is an error.
+
+### Blink > Network
+
+* The [cache mode](https://fetch.spec.whatwg.org/#concept-request-cache-mode) of a `Request` can now be specified using
+  the [`cache`](https://fetch.spec.whatwg.org/#dom-request-cache) option.
+* The cache mode of a `Request` can now be accessed using
+  [`Request.prototype.cache`](https://developer.mozilla.org/en-US/docs/Web/API/Request/cache). Reload requests, the
```
[the remark this answers is not in the document]
Comment (fsoder) [reply src/blog/_posts/2018-01-31-opera-51.md]: Updated along those lines.

**Pull request merged**